Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM lambci/lambda:build-nodejs12.x
+
+WORKDIR /tmp
+
+COPY . .
+
+RUN npm pack --unsafe-perm
```

Comment:
when running in a Docker container as root, this flag is necessary to force `prepare` to run, which in our config runs `build`.